scripts/getTask.ts

```diff
@@ -1,16 +1,17 @@
 import { ethers } from "hardhat";
 import abi from "../artifacts/contracts/Web3Task.sol/Web3Task.json"
 
+const { CONTRACT_ADDRESS } = process.env;
+
 async function main() {
     const [signer] = await ethers.getSigners();
 
-    const contract = new ethers.Contract("0xec20dcbf0380f1c9856ee345af41f62ee45a95a1", abi.abi, signer);
-
-    const getTaskResponse = await contract.getTask(0);
-
-
-    console.log('GetTaskResponse = ', getTaskResponse);
+    const contract = new ethers.Contract(`${CONTRACT_ADDRESS}`, abi.abi, signer);
 
+    for (let i = 0; i <= 50; i++) {
```

Comment:
The getTask shouldn't have a for loop. We currently have a script for multiCall precisely to cover this need.

Reply:
ok